cpp/foxglove-websocket/include/foxglove/websocket/websocket_client.hpp

```diff
@@ -122,7 +122,8 @@ class Client : public ClientInterface {
       return;  // Already disconnected
     }
 
-    _endpoint.close(_con, websocketpp::close::status::going_away, "");
+    websocketpp::lib::error_code ec;
```

Comment:
what's the point of this variable? does it prevent throwing an exception? Might be nice to add a comment to explain this

Reply:
Yeah, it prevented throwing an exception when calling `close` multiple times. Will add a comment